README.md

```diff
@@ -1,3 +1,164 @@
-# OpenSlides client
+# OpenSlides Client
```

Comment:
OpenSlides client or OpenSlides Client? @emanuelschuetze

Reply:
Let us use always capital letters for service names! As in our [architecture diagram](https://raw.githubusercontent.com/OpenSlides/OpenSlides/openslides4-dev/docs/OpenSlides4-systemarchitecture.png). E.g. "OpenSlides Backend Service", not "OpenSlides backend service".

Reply:
Ok.